mappings/net/minecraft/block/MapColor.mapping

```diff
@@ -0,0 +1,35 @@
+CLASS net/minecraft/class_259 net/minecraft/block/MapColor
+	FIELD field_2737 COLORS [Lnet/minecraft/class_259;
+	FIELD field_2738 CLEAR Lnet/minecraft/class_259;
+		COMMENT RGB: 0, 0, 0
+	FIELD field_2739 PALE_GREEN Lnet/minecraft/class_259;
+		COMMENT RGB: 127, 178, 56
+	FIELD field_2740 PALE_YELLOW Lnet/minecraft/class_259;
+		COMMENT RGB: 247, 233, 163
+	FIELD field_2741 LIGHT_GRAY Lnet/minecraft/class_259;
+		COMMENT RGB: 167, 167, 167
+	FIELD field_2742 RED Lnet/minecraft/class_259;
+		COMMENT RGB: 255, 0, 0
+	FIELD field_2743 LIGHT_BLUE Lnet/minecraft/class_259;
+		COMMENT RGB: 160, 160, 255
+	FIELD field_2744 LIGHT_GRAY2 Lnet/minecraft/class_259;
```

Comment:
Since there are two `LIGHT_GRAY`s, with one being for Sponge, Wool, and Cobweb and the other being for Metal blocks, does it make sense to use the associated material types in the color names for `LIGHT_GRAY` and `LIGHT_GRAY2`? The first light gray can probably stay as-is since it's a bit more multi-purpose, but maybe `METAL_GRAY` instead of `LIGHT_GRAY2`?